test/base/SpecBase.scala

```diff
@@ -251,7 +252,7 @@ trait SpecBase extends AnyWordSpec with Matchers with GuiceOneAppPerSuite {
         summary = LSPSummary(
           activePenaltyPoints = 10,
```

Comment:
If there is only one LSP here, it is worth changing this to activePenaltyPoints = 1

Reply:
Yes, Done